### DISCUSSION TITLE
Applying style to multiple tag types?

### DISCUSSION THREAD

**Discussion by rachelmflowers** [jacksleight/statamic-bard-texstyle#32] [General]
I want to create an Accessible headings style dropdown for my clients so they can choose to override tags, allowing them to choose the correct heading or tag semantically to keep the proper order for accessibility, but still have control over the style of that heading.

I've done this in tinyMCE by configuring a list of acceptable selectors such as:
```           
{title: 'Headline Sizes', items: [
    {title: 'Heading XXXL', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-xxxl'},
    {title: 'Heading XXL', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-xxl'},
    {title: 'Heading XL', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-xl'},
    {title: 'Heading L', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-l'},
    {title: 'Heading M', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-m'},
    {title: 'Heading S', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-s'},
    {title: 'Heading XS', selector: 'p,span,h1,h2,h3,h4,h5,h6', classes: 'header-xs'},
]},
```

Is there a way that I can do that with this addon as well? The documentation made it seem like I was only able to select a single selector to apply the class names to.

**Comment by jacksleight** [comment]
> The documentation made it seem like I was only able to select a single selector to apply the class names to.

Yes, currently each style has to target a specific type (and level for headings). However this is really just a hangover from Tiptap 1 (which Statamic <= 3.3 used) as that was far more restrictive than v2. It should be possible to implement this kind of functionality now, and I think it would be a nice option to have.

I've created an issue for it: https://github.com/jacksleight/statamic-bard-texstyle/issues/33.

In the meantime you could use the attributes panel instead, by adding a class field to the relevant types and using that to set the class. Not as simple as the toggle buttons but it would achieve the same end result.

**Comment by jacksleight** [comment]
This was added in https://github.com/jacksleight/statamic-bard-texstyle/releases/tag/3.5.0
